### ISSUE TITLE
Release in public Maven Repository

### ISSUE THREAD

**Issue by blaubaer** [neilalexander/jnacl#17]
Hi!

What do you think about to release it on the public Maven Repository?

Cheers!

**Comment by neilalexander** [comment]
Hi! I was under the impression that it may already be on some public Maven repositories available for use, thanks to the ZeroMQ project. Did you have anywhere specific in mind?

**Comment by blaubaer** [comment]
In the `pom.xml` are the required entries listed like `https://oss.sonatype.org/service/local/staging/deploy/maven2/` etc. But if I go to http://search.maven.org/ and search for `com.neilalexander` there is nothing listed. If I search for `jnacl` I found a version of ZeroMQ (correct) but this is quite old (`29-Jun-2017`) and does not reflect your latest changes.
Maybe do it by your own? Normally this is quite simple. Just follow the instructions listed here: http://central.sonatype.org/pages/ossrh-guide.html
Cheers!

**Comment by neilalexander** [comment]
I’ll be glad to take a look certainly!

**Comment by rlenferink** [comment]
Hi @blaubaer 

We just released this to Maven Central as version 1.0.0
It can be used in your project by including the following dependency.

```text
<dependency>
    <groupId>eu.neilalexander</groupId>
    <artifactId>jnacl</artifactId>
    <version>1.0.0</version>
</dependency>
```

If you're having any questions or remarks, just let us know.

**Comment by blaubaer** [comment]
That is a great success. Thank you! 👍
